### ISSUE TITLE
Determining Which Crossing Line Triggered

### ISSUE THREAD

**Issue by pkscout** [hjdhjd/homebridge-unifi-protect#1083]
### Describe the feature request

I would like to be able to tell which crossing line triggered.  I can see in the UniFi console when a recording was triggered by a crossing line (although not which crossing line, at least not in the UniFi console UI).

### Describe the proposed solution

There may not be a solution.  This is similar to:  https://github.com/hjdhjd/homebridge-unifi-protect/issues/992

I'm hoping perhaps the API has matured some since that issue was opened and that it might now be possible to have a contact sensor for each crossing line.

### Describe alternatives you have considered to the enhancement

I'm really trying to determine occupancy in a space, but if I walk in and sit down, the motion sensor and then occupancy sensor turn off after not very long, and I don't want to artificially set a super long time for occupant, as the lights will then stay on for way longer than they should.  I've tried mmWave sensors, but none of them seem to want to see me sitting on the couch either, and the space is too open to use a time of flight threshold sensor.  I'm trying to use the crossing lines as a person counter so that I can count when someone enters and exits the space (I have two crossing lines, one A>B and one B>A, so I can, in theory, tell the difference between an entry and an exit).

**Comment by hjdhjd** [comment]
Don’t believe that telemetry’s available, unfortunately…I’ll take a closer look, but I haven’t seen it. The occupancy sensor capabilities are really the best route to do this, in general…along with the highest sensitivity setting to any motion in a room.

If you can’t get purpose-built occupancy sensors to do the job that you want…not sure what else to tell you. 😄 It seems to me that something like the Aqara FP2 is more than capable of this.

**Comment by pkscout** [comment]
Thanks. Out of frustration I got my FP2s back out and tested them in various places in the room. I finally got some placements that work better (no false negatives, some false positives), so I'm back to using those. I still like threshold sensing because it's "stateless" once you're in a space, but until the Ubiquity API is more verbose, this will do. 

I appreciate you responding and looking into this.
